cypress/integration/group2/myworkflows.ts

```diff
@@ -108,24 +107,37 @@ describe('Dockstore my workflows', () => {
       cy.contains('1 – 2 of 2');
       cy.contains('Close').click();
     });
-    it('Should contain the extended properties and be able to edit the info tab', () => {
+    it.only('Should contain the extended properties and be able to edit the info tab', () => {
```

Comment:
Should remove `.only` so other tests aren't ignored

Reply:
Ugh, thanks for catching that.